### ISSUE TITLE
SmartEnum causes error

### ISSUE THREAD

**Issue by djek-sweng** [PawelGerr/Thinktecture.Runtime.Extensions#5]
Hi Pawel,

using updated version <PackageReference Include="Thinktecture.Runtime.Extensions" Version="4.1.0" /> now causes this error:

'IEnumerable<MySmartEnum>' does not contain a definition for 'ToList' and no accessible extension method 'ToList' accepting a first argument of type 'MySmartEnum<Gender>' could be found (are you missing a using directive or an assembly reference?)

````
using Thinktecture;

namespace MyNamespace;

public partial class MySmartEnum : IEnum<string>
{
   public static readonly MySmartEnum MyItem = new("MyItem");
}
````

See MyNamespace.MySmartEnum.g.cs:

````
/* file: MyNamespace.MySmartEnum.g.cs */

/// <summary>
/// Gets all valid items.
/// </summary>
public static global::System.Collections.Generic.IReadOnlyList<global::MyNamespace.MySmartEnum> Items => _items ??= ItemsLookup.Values.ToList().AsReadOnly();
````

Greetings,
David

**Comment by PawelGerr** [comment]
You're right, with `ImplcitUsings` enabled, the missing namespaces stay unnoticed.
I have disabled `ImplicitUsing` in my tests and samples, so I see that something is missing right away.

I released 4.1.1

**Comment by djek-sweng** [comment]
Works fine. Thank you.
